File: Initial Prototype/Client/MyApplicationProject/MyApplication/src/main/java/com/example/myapplication/dtos/LoadRangeDTO.java
package com.example.myapplication.dtos;

/**
 * Created by Michal on 17/01/14.
 */
public class LoadRangeDTO {

    private int Id;
    private int Take;
    private int Skip;

    public LoadRangeDTO(int id, int skip, int take)
    {
        this.Id = id;
        this.Skip = skip;
        this.Take = take;
    }
}

File: Initial Prototype/ServerSide/FindNDrive/FindNDriveServices2/DTOs/LoadRangeDTO.cs
// --------------------------------------------------------------------------------------------------------------------
// <copyright file="LoadRangeDTO.cs" company="">
//   
// </copyright>
// <summary>
//   The load range dto.
// </summary>
// --------------------------------------------------------------------------------------------------------------------

namespace FindNDriveServices2.DTOs
{
    using System.Runtime.Serialization;

    /// <summary>
    /// The load range dto.
    /// </summary>
    [DataContract]
    public class LoadRangeDTO
    {
        /// <summary>
        /// Gets or sets the id.
        /// </summary>
        [DataMember]
        public int Id { get; set; }

        /// <summary>
        /// Gets or sets the skip.
        /// </summary>
        [DataMember]
        public int Skip { get; set; }

        /// <summary>
        /// Gets or sets the count.
        /// </summary>
        [DataMember]
        public int Take { get; set; }
    }
}